README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build status](https://dev.azure.com/github/Atom/_apis/build/status/Atom%20Production%20Branches?branchName=master)](https://dev.azure.com/github/Atom/_build/latest?definitionId=32&branchName=master)
 
-> Atom and all repositories under Atom will be archived on December 15, 2022. Learn more in our [official announcement](https://github.blog/2022-06-08-sunsetting-atom/)
+> Due to changes in the upstream, Atom and all it's repositories under the original Atom branding will be archived on December 15, 2022. Get involved here while we work on the future of Atom-community. You can learn more in their [official announcement](https://github.blog/2022-06-08-sunsetting-atom/) and get involved here.
```

Comment:
```suggestion
> Due to upstream changes, [Atom and all repositories under the original Atom branding will be archived on December 15, 2022](https://github.blog/2022-06-08-sunsetting-atom/). Get involved here and at [the discord](https://discord.gg/deuBJaqVud) while we work on the future of Atom-community.
```

I think a commit can include `[skip ci]` in the name to avoid testing